### DISCUSSION TITLE
Welcome to GridPath Discussions!

### DISCUSSION THREAD

**Discussion by anamileva** [blue-marble/gridpath#1051] [Announcements]
## Welcome to GridPath Discussions!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Share resources.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together.

**Comment by sriharid** [comment]
Great to see this. Hope this helps the community grow, and look forward to rich discussions.

**Comment by MahavirSingh20** [comment]
I have been working on GridPath for a few months. Recently came across a question on the operations of Hydro plant and Pumped storage plants. Does a Pumped storage plant have the same PLF as Hydro plants (generally around 40%)? Kindly provide some inputs and if possible a relevant reference.
